src/core/public/application/application_service.test.tsx

```diff
@@ -17,19 +17,57 @@
  * under the License.
  */
 
+import { basePathServiceMock } from '../base_path/base_path_service.mock';
+import { injectedMetadataServiceMock } from '../injected_metadata/injected_metadata_service.mock';
+import { MockCapabilitiesService } from './application_service.test.mocks';
 import { ApplicationService } from './application_service';
 
-describe('#setup()', () => {
-  it('exposes mounting interface', () => {
-    const applicationService = new ApplicationService();
-    const satupApplicationService = applicationService.setup();
-
-    expect(satupApplicationService).toHaveProperty('mount');
-
-    const mountHandler = jest.fn();
+describe('#start()', () => {
+  it('exposes available apps from capabilities', async () => {
+    const service = new ApplicationService();
+    const setup = service.setup();
+    setup.registerApp({ id: 'app1' } as any);
+    setup.registerLegacyApp({ id: 'app2' } as any);
+    const injectedMetadata = injectedMetadataServiceMock.createStartContract();
+    const basePath = basePathServiceMock.createStartContract();
+    expect((await service.start({ basePath, injectedMetadata })).availableApps)
+      .toMatchInlineSnapshot(`
```

Comment:
Can you explain why using this inline snapshot is preferable to just using a `toEqual`?

```js
expect((await service.start({ basePath, injectedMetadata })).availableApps)
  .toEqual([
    { id: 'app1' },
    { id: 'app2' },
  ]);
```

Reply:
Mostly just that I'm lazy and this is a bit faster than typing that out. [IMO this is a good use of snapshots](https://www.joshdover.com/when-not-to-use-jest-snapshots/), but I don't think we have any Kibana-wide consensus on snapshots.